cmd/kubeadm/app/cmd/config.go

```diff
@@ -449,9 +449,17 @@ func NewImagesPull(runtime utilruntime.ContainerRuntime, images []string) *Image
 	}
 }
 
-// PullAll pulls all images that the ImagesPull knows about
-func (ip *ImagesPull) PullAll() error {
+// PullControlPlaneImages pulls all images that the ImagesPull knows about
+func (ip *ImagesPull) PullControlPlaneImages() error {
 	for _, image := range ip.images {
+		ret, err := ip.runtime.ImageExists(image)
```

Comment:
The `PullImage` call should automatically do this and not return an error if image exists. Hence, let's remove this call.

Reply:
removed it